coverage.json

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
Did you intend to commit this?

Reply:
no, I'll remove it and add it to gitignore